gwcli/storage.py

```diff
@@ -76,7 +76,7 @@ def _get_disk_meta(self, cluster_ioctx, disk_meta):
                 disk_meta[rbd_name] = {}
                 with cluster_ioctx.open_ioctx(pool) as ioctx:
                     try:
-                        with rbd.Image(ioctx, image) as rbd_image:
+                        with rbd.Image(ioctx, image, read_only=True) as rbd_image:
```

Comment:
I can't seem to replicate the bug that this was fixing. Can you tell me how you are confirming the disk is full?